src/js/tech/html5.js

```diff
@@ -105,29 +91,27 @@ class Html5 extends Tech {
         const clone = el.cloneNode(false);
         Html5.disposeMediaElement(el);
         el = clone;
-        player.tag = null;
       } else {
         el = Lib.createEl('video');
 
         // determine if native controls should be used
-        let attributes = VjsUtil.mergeOptions({}, player.tagAttributes);
-        if (!Lib.TOUCH_ENABLED || player.options()['nativeControlsForTouch'] !== true) {
+        let tagAttributes = this.options_.tag && Lib.getElementAttributes(this.options_.tag);
+        let attributes = VjsUtil.mergeOptions({}, tagAttributes);
+        if (!Lib.TOUCH_ENABLED || this.options_.nativeControlsForTouch !== true) {
           delete attributes.controls;
         }
 
         Lib.setElementAttributes(el,
           Lib.obj.merge(attributes, {
-            id: player.id() + '_html5_api',
```

Comment:
We should just have the player generate the full ID when it's building the tech.